README.md

```diff
@@ -78,6 +78,23 @@ addDecorator(withIntl);
 configure(() => require('./stories'), module);
 ```
 
+Example config for `react-intl` V3:
+```
+import enLocaleData from '<assets>/lang/en.json'
```

Comment:
I think we should try to avoid the import of the messages as `localeData` as it's confusing how this name is used in the V2 example.

We should stick as much as possible to the example for V2 and use a message object instead of imports.

Reply:
Will change it when have time. Sorry for later reply.

Reply:
@floriangosse I updated description.